easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb

```
@@ -0,0 +1,3525 @@
+    }),
+    ('aplot', '0.1.10', {
+        'checksums': ['d937768241f887628b88bb3b49dd6cbe9b7dae39ae7054e7380a9836721a67d1'],
+    }),
+    ('tidytree', '0.4.2', {
+        'checksums': ['cb831a66d8afa5e21f5072e4fbebcbd2228881090d0040f87605f5aeefda155e'],
+    }),
+    ('ggvenn', '0.1.9', {
+        'checksums': ['38779cfa4e01e07d4a42453cfb8b93d32d2acd6187676bae7d11b2168714a52e'],
+    }),
+    ('scatterpie', '0.1.8', {
+        'checksums': ['a6ccc63a8be63fa113704cf5d4893c1ec1b75d3081ab971bd70e650e708872a0'],
+    }),
+    ('shadowtext', '0.1.2', {
+        'checksums': ['253c4e737dbb302aa0729e5074e84cbfde2a73bfd7a0fd2c74b557cb728bae7d'],
+    }),
+    ('random', '0.2.6', {
+        'checksums': ['2b59f9bce0c3ebf8215ab42dffaf9a1c7eea7468964063215a8d422af953b069'],
+    }),
+    ('R2WinBUGS', '2.1-21', {
+        'checksums': ['fa86cb5140f5dfce29f9517b94d6d08c316ddcffa4aa3085945b84d34910134d'],
+    }),
+    ('aricode', '1.0.2', {
+        'checksums': ['5f4e51f7063cfd4924a8925c902a3eb7517c743234a14560691cb84428b6b2f0'],
+    }),
+    ('DepthProc', '2.1.5', {
+        'checksums': ['e6b0afd54bb20e25a6bf5402c771848db20e9c844f0fc990ecc3d1078b9eee44'],
+    }),
+    ('dbscan', '1.1-11', {
+        'checksums': ['f0498e67e612629340a2758fbe747c4d9d4ca648f002230a03499cb73735e62f'],
+    }),
+    ('ggh4x', '0.2.3', {
+        'checksums': ['8af0e855bf221ce53b3162ae9d25a8d04525aabbda2f63c4843d86ee3eca4535'],
+    }),
+    ('ComplexUpset', '1.3.3', {
+        'checksums': ['5b2f99b4a38648641c7d31fc57f201a93e5bc1b85442a0b9726f72c166d964ea'],
+    }),
+    ('proxyC', '0.3.3', {
+        'checksums': ['3e25da94633da0b5c6cdac03f5b6d8205d078f60c5a09073e90a1b6f0930e302'],
+    }),
+    ('changepoint', '2.2.4', {
+        'checksums': ['ac636fde7610137385dde1e3d8a22a2ff856a8d5c917c7ad1a5cc49f98b8649b'],
+    }),
+    ('geeM', '0.10.1', {
+        'checksums': ['fe76a32981b55835095041e777d1cf2e9ce43edb8d9488db56279f7cb6f43fe5'],
+    }),
+    ('ggstance', '0.3.6', {
+        'checksums': ['cd9ad4fb83c583009ee45371c7d02d32b33d06238d3f0162b3ba9851e27a6372'],
+    }),
+    ('mosaicCore', '0.9.2.1', {
+        'checksums': ['b7ce16f20fde1d8a5b1836110e5980fb274504ddae6b9c5c2f4198dc7d09dafa'],
+    }),
+    ('ggformula', '0.10.2', {
+        'checksums': ['948b1c34a5c089cf86275d2b3e020e1a099278136e43175a50fb6063e54b6375'],
+    }),
+    ('kinship2', '1.9.6', {
+        'checksums': ['0150bd5974c2a19885f6ff2e99f3c1f6361054a0910dcfc9dd26a8e0cd73bbf6'],
+    }),
+    ('MESS', '0.5.9', {
+        'checksums': ['b57dff943a130830a24d87fe2e8f8b766cf18398c9c09e0a020d5dd183f9d72d'],
+    }),
+    ('smoof', '1.6.0.3', {
+        'checksums': ['af8664b152876c545f6545528de6e05a289d0714103fac7afc52960a9a855fb1'],
+    }),
+    ('mlrMBO', '1.1.5.1', {
+        'checksums': ['0cf26e5e9b180d15b932541cf081a552703a60edf762aafca9933c24ea91dc99'],
+    }),
+    ('emoa', '0.5-0.1', {
+        'checksums': ['bff6c84274e9bdcbdb82d6edd90265844f7cd23f22abbf68d6b563ef48237966'],
+    }),
+]
```

```suggestion
    ('webutils', '1.1', {
        'checksums': ['33e2b408d2b9ca12e158e0007d4a55a3ace6838341ad944344de2ceb806e4699'],
    }),
    ('swagger', '3.33.1', {
        'checksums': ['528369b04c6142112e05ee8b950ca1c01b3e390a4515eacb0fa17db98bafafd1'],
    }),
    ('varhandle', '2.0.5', {
        'checksums': ['9b0ee653e0343e292547d2a7052e60a2e7d97d1d5528246862522e67346882d0'],
    }),
]
```

These got added in #17525

---

done in f62f8d8